config/db.js

```diff
@@ -1,10 +1,8 @@
 require('dotenv').config();
 const mongoose = require('mongoose');
 
-const db =
-  process.env.MONGO_URL ||
-  'mongodb+srv://lv487nodejs:nodejslv487@cluster0-ltcgb.mongodb.net/horondi?retryWrites=true&w=majority';
-
+const db =  process.env.MONGO_URL
+  || 'mongodb+srv://user-read-write1:sfd$42A3a@horondi-yl27g.mongodb.net/horondi-3?retryWrites=true&w=majority';
```

Comment:
all references to the database should be at .env file. not in configs

Reply:
fixed